VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/waiting-room-shared/tests/waiting-room-base.component.non-event.spec.ts

```diff
@@ -94,6 +94,47 @@ describe('WaitingRoomComponent message and clock', () => {
         videoWebService.getConferenceById.calls.reset();
     });
 
+    describe('toggle Panel', () => {
+        const participantPanelName = 'Participants';
+        const chatPanelName = 'Chat';
+
+        it('should switch Participants from false to true  when starting from false', () => {
+            // Arrange
+            component.panelStates[participantPanelName] = false;
+
+            // Act & Assert
+            component.togglePanel(participantPanelName);
+            expect(component.panelStates.Participants).toBe(true);
+        });
```

Comment:
I think it would be worth having a test for chat going from true to false either in here or in a separate test with the same setup.

Reply:
```suggestion
        it('should switch Participants from false to true and all others to false when starting from false', () => {
            // Arrange
            component.panelStates[participantPanelName] = false;
            component.panelStates[chatPanelName] = true;

            // Act & Assert
            component.togglePanel(participantPanelName);
            expect(component.panelStates[participantPanelName]).toBe(true);
            expect(component.panelStates[chatPanelName]).toBe(false);
        });
```

Reply:
added additional assert